### ISSUE TITLE
[ccxt]  Bot stops on /status table request aparently because of empty ticker

### ISSUE THREAD

**Issue by botum** [gcarq/freqtrade#613]
@# Step 1: Have you search for this issue before posting it?

Yes, no report yet.

## Step 2: Describe your environment

  * Python Version: 3.6
  * Branch: feat/objectify-ccxt
  * Last Commit ID: HEAD
 
## Step 3: Describe the problem:

While dry-running the bot stops with exception.

### Steps to reproduce:

  1. dry-run
  2. may be related to status request or trade selling
  3. bot stops
  
### Observed Results:

  * What happened?
  * What did you expect to happen?

### Relevant code exceptions or logs:
```
Status: OperationalException:
Traceback (most recent call last):
  File "/var/local/user/freqtrade/freqtrade/exchange/__init__.py", line 262, in get_ticker_history
    return _API.fetch_ohlcv(pair, timeframe=tick_interval)
  File "/var/local/user/miniconda3/lib/python3.6/site-packages/ccxt/bittrex.py", line 424, in fetch_ohlcv
    raise ExchangeError(self.id + ' returned an empty or unrecognized response: ' + self.json(response))
ccxt.base.errors.ExchangeError: bittrex returned an empty or unrecognized response: {"success":true,"message":"","result":null}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/local/user/freqtrade/freqtrade/freqtradebot.py", line 162, in _process
    state_changed = self.process_maybe_execute_buy()
  File "/var/local/user/freqtrade/freqtrade/freqtradebot.py", line 340, in process_maybe_execute_buy
    if self.create_trade():
  File "/var/local/user/freqtrade/freqtrade/freqtradebot.py", line 285, in create_trade
    (buy, sell) = self.analyze.get_signal(_pair, interval)
  File "/var/local/user/freqtrade/freqtrade/analyze.py", line 127, in get_signal
    ticker_hist = get_ticker_history(pair, interval)
  File "/var/local/user/freqtrade/freqtrade/exchange/__init__.py", line 35, in wrapper
    return f(*args, **kwargs)
  File "/var/local/user/freqtrade/freqtrade/exchange/__init__.py", line 268, in get_ticker_history
    raise OperationalException('Could not fetch ticker data. Msg: {}'.format(e))
freqtrade.OperationalException: Could not fetch ticker data. Msg: bittrex returned an empty or unrecognized response: {"success":true,"message":"","result":null}

Issue /start if you think it is safe to restart.
```

**Comment by botum** [comment]
Happens on status request.

**Comment by xmatthias** [comment]
Oddly enough, it seems to work for me (now). 
Does this reliably each time you run `/status`?

could be related to unexpected API results (similar to #614 ) ...

**Comment by siedi** [comment]
Just to confirm, this is exactly one of the errors I get regularly (in not related to /status), see my comment in #614

**Comment by botum** [comment]
Right, not related to status, but, maybe when many request "stranger" things happen to this exchange.

**Comment by xmatthias** [comment]
i think gracefully handling errors from the get_ticker / get_ticker_history functions should be the way to go - different exchanges (especially smaller ones) may not have a very stable API.

We should however also check if we run into rate-limiting, which could also explain some of the observed errors.

**Comment by gcarq** [comment]
Rate limite is now enabled on branch `feat/objectify-ccxt`, that should improve the stability a little bit. I will open a PR for enhanced exception handling, which should fix this issue.

**Comment by gcarq** [comment]
Should be fixed on `feat/objectify-ccxt` at a5c1547251b5ff0ba10153ff26f639e17281be3a
